lda-data.h

```diff
@@ -6,7 +6,7 @@
 
 #include "lda.h"
 
-#define OFFSET 0;                  // offset for reading data
+static const int OFFSET = 0;  // offset for reading data
```

Comment:
this value is only used in one place, and it is zero... not sure what value there is defining it at all